packages/web/src/components/gcds-checkbox/stories/gcds-checkbox.stories.tsx

```diff
@@ -145,9 +145,102 @@ const Template = (args) => (`
 </GcdsCheckbox>
```

Comment:
Can't comment on the line directly, but line 142 has a typo. It says `validate)n` instead of `validateOn`.